### PR TITLE
fix: homepage ux v2

### PR DESCRIPTION
**Pull request by rcmarron**
## Problem
There were a bunch of UX loose ends on the homepage + primary dashboard work. This PR cleans them up, so they match [figma](https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=7868%3A68091)

## Changes

* Dashboard list shows the home icon
* Dashboard list item menu has link to set primary dashboard on home page
* If dashboards have long descriptions, they're truncated in the primary dashboard modal
* Homepage dashboard header doesn't jump on loading (and icon isn't shown until dashboard loads)

https://user-images.githubusercontent.com/4813045/157566414-2dacbf0d-c5b0-484d-ab0a-124abf5e1bbb.mov

👉 *Stay up-to-date with our [coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

Switched the primary dashboard and made sure the things mentioned above were correct

### CODE REVIEW

File: frontend/src/scenes/dashboard/Dashboards.tsx [hunk 1]
```diff
@@ -58,6 +62,7 @@ export function Dashboards(): JSX.Element {
             dataIndex: 'name',
             width: '40%',
             render: function Render(name, { id, description, _highlight, is_shared }) {
+                const is_primary = id === currentTeam?.primary_dashboard
```
Comment (liyiy) [inline frontend/src/scenes/dashboard/Dashboards.tsx:65]:
```suggestion
                const isPrimary = id === currentTeam?.primary_dashboard
```

File: frontend/src/scenes/dashboard/Dashboards.tsx [hunk 2]
```diff
@@ -69,6 +74,11 @@ export function Dashboards(): JSX.Element {
                                     <ShareAltOutlined style={{ marginLeft: 6 }} />
                                 </Tooltip>
                             )}
+                            {is_primary && (
```
Comment (liyiy) [inline frontend/src/scenes/dashboard/Dashboards.tsx:77]:
```suggestion
                            {isPrimary && (
```

**Review by liyiy — APPROVED** [review]
LGTM, I'd just wait for @clarkus to review the UI look on things

File: frontend/src/scenes/dashboard/Dashboards.tsx (hunk 1 above)
Comment (rcmarron) [reply frontend/src/scenes/dashboard/Dashboards.tsx]: Ooo good catch!

**Review by alexkim205 — APPROVED** [review]
nice!

**Review by clarkus — APPROVED** [review]

**Pull request merged**